go/ir/derivation.go

```diff
@@ -277,7 +262,7 @@ func getMaxIndexOfKeyValueData(str string) (int, error) {
 
 func fillFieldDesc(columnTypeList []*sql.ColumnType, rowdata []interface{}, fieldDescMap FieldDescMap, rowCount int, originalSizes map[string]int) error {
 	for idx, ct := range columnTypeList {
-		_, fld := decomp(ct.Name())
+		_, fld := verifier.Decomp(ct.Name())
```

Comment:
`Decomp` is sort of database functionality, put it to database package should be more reasonable.